README.md

```diff
@@ -1 +1,41 @@
-# leaderboard
\ No newline at end of file
+# leaderboard
+
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+# Built With
+
+- HTML
+- CSS
+- WEBPACK
+# Getting started
+
+## To get a local copy of this repository kindly follow the steps below.
+- Scroll to top of this current repository
+- Click on the `Code` button with background color green on the right end corner
+- Click on the clipboard icon on the extreme right of the dropdown to copy the repository link
+- In your local PC, open your terminal or command prompt in the folder you would like to clone this repository into
+- Type `git clone (copied link)` on the currently opened terminal or command prompt
+- Remember to change `(copied link)` to `https://github.com/TSHEPO-CLOUD/leaderboard/tree/project-setup` which is the name of the repositor
+
+#INSTALL
+Run `npm install`
+Run `npm build`
+
+## Contributors
```

Comment:
**Kindly work on few changes:**

- PROVIDE A LICENSE.
- Your README file should have instructions about where and how to generate any necessary API keys, credentials, how to set, store, and generate them.